src/app/errors/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react';
+import { connect } from 'react-redux';
+import Sentry from '../../../helpers/Sentry';
+import { Dispatch } from '../../types';
+import { recordError } from '../actions';
+
+interface Props {
+  children: ReactNode;
+  recordError: (error: Error) => void;
+}
+
+class ErrorBoundary extends React.Component<Props> {
+
+  public componentDidCatch(error: Error) {
+    this.props.recordError(error);
+    // also log the error to an error reporting service
+    Sentry.captureException(error);
```

Comment:
probably put this line above the call to `recordError` to make sure it goes out before anything else might break